src/index.js

```diff
@@ -1,50 +1,71 @@
 import './style.css';
+import localstorage from './localstorage.js';
+import userInteraction from './userInteraction.js';
 
-const todoData = [
-  {
-    description: 'Doing house chores',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Joining Microverse class',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Taking a nap',
-    completed: false,
-    index: 0,
-  },
-];
-
-todoData.forEach(() => {
-  for (let i = 0; i < todoData.length - 1; i += 1) {
-    if (todoData[i].index > todoData[i + 1].index) {
-      const temp = todoData[i];
-      todoData[i] = todoData[i + 1];
-      todoData[i + 1] = temp;
+/* eslint max-classes-per-file: ["error", 3] */
+
+class List {
+  constructor(description, index) {
+    this.description = description;
+    this.index = index;
+  }
+}
+
+const { UI } = userInteraction;
+const { StorageInLocal } = localstorage;
+const store = new StorageInLocal();
+const storeArray = store.getLists();
+
+storeArray.map(() => {
+  for (let i = 0; i < storeArray.length - 1; i += 1) {
+    if (storeArray[i].index > storeArray[i + 1].index) {
+      const temp = storeArray[i];
+      storeArray[i] = storeArray[i + 1];
+      storeArray[i + 1] = temp;
     }
   }
+  return null;
+});
+
+document.addEventListener('DOMContentLoaded', UI.displayAllLists);
+document.querySelector('#addForm').addEventListener('submit', (e) => {
```

Comment:
-  [OPTIONAL] I highly recommend that enabling the add item button to add tasks will improve the user experience, currently, items are added only by pressing the "Enter".

![image](https://user-images.githubusercontent.com/68030297/229305980-4736f968-adf9-4af5-be1d-3c58952f334d.png)